icu_base.py

```diff
@@ -103,8 +97,9 @@ def build(self):
         if tools.is_apple_os(self._the_os):
             self._env_build.defines.append("_DARWIN_C_SOURCE")
             if self.settings.get_safe("os.version"):
-                self._env_build.flags.append(tools.apple_deployment_target_flag(self._the_os,
-                                                                            self.settings.os.version))
+                self._env_build.flags.append(
```

Comment:
Please do me a favor next time, and put the reformatting of code into separate commits. Mixing this up means that I have to eyeball changes in the commit that have nothing to do with Solaris, figure out that they're only formatting changes. I might be doing this with differencing tools that aren't as sophisticated as on Github.

If there are separate commits, I'm happy enough stepping through them.

Reply:
Sorry about that - will do.